tests/installation/installation_overview.pm

```diff
@@ -41,6 +41,22 @@ sub ensure_ssh_unblocked {
                 send_key_until_needlematch 'ssh-open', 'tab';
             }
         }
+        #performance ci need diable firewall
+        if (get_var('PERF_KERNEL')) {
+            send_key_until_needlematch [qw(firewall-enable firewall-disable)], 'tab';
+            if (match_has_tag 'firewall-enable') {
+                if (get_var('FIREWALL')) {
```

Comment:
You want to disable firewall, then the setting is better to be DISABLE_FIREWALL, than only FIREWALL?

Reply:
Updated ,change to "DISABLE_FIREWALL"